difflint/lib.py

```diff
@@ -249,7 +197,25 @@ def detect_new_diff_lint_errors(log_output):
 
 
 def main():
-    if not lint_script_exists() or not all_files_configured():
+    parser = argparse.ArgumentParser(description='Linter that will examine ' +
+        'only new changes as you commit them.')
+    parser.add_argument('-c', '--check', action='store_true',
```

Comment:
I'd suggest consistently exiting with an error if linters are missing, whether `--check` is passed or not. The `--check` flag would perform the check and exit immediately with 0 or an error code, so that if you wanted to ignore that error, you could do so without performing the expensive linting operation.

More long term, I think as more linters are added it doesn't make as much sense to check whether they are all present. If you're only writing Racket code, for example, you don't care if JSCS is installed. I don't know what a good way of implementing this is, though.